extensions/extensions-data-lineage/extensions-data-lineage-http-consumer-service/src/test/resources/application.properties

```diff
@@ -9,5 +9,6 @@
 ###
 quarkus.rest-client."com.boozallen.aissemble.datalineage.consumer.HttpProducerService".uri=http://127.0.0.1:15100/endpoint
 quarkus.amqp.devservices.image-name=quay.io/artemiscloud/activemq-artemis-broker:1.0.26
+quarkus.kafka.devservices.image-name=vectorized/redpanda:v24.1.7
```

Comment:
Any reason not to use `vectorized/redpanda:latest`?

Reply:
Generally, we want to avoid LATEST b/c it normally just means "most recently pushed" vice "newest release".  Also better for version control to know what we depend one (Python 'best practices' withstanding)